client/app/states/templates/editor/editor.state.js

```diff
@@ -0,0 +1,40 @@
+/* eslint camelcase: "off" */
+
+/** @ngInject */
+export function TemplatesEditorState(routerHelper) {
+  routerHelper.configureStates(getStates());
+}
+
+function getStates() {
+  return {
+    'templates.editor': {
+      url: '/edit/:templateId',
+      templateUrl: 'app/states/templates/editor/editor.html',
```

Comment:
Thinking importing the html template should be done explicitly (`import` at the header of the file)